flutter/ios/Classes/SentryFlutterPluginApple.swift

```diff
@@ -519,6 +519,9 @@ public class SentryFlutterPluginApple: NSObject, FlutterPlugin {
         if let extras = user["extras"] as? [String: Any] {
           userInstance.data = extras
         }
+        if let data = user["data"] as? [String: Any] {
+          userInstance.data = data
+        }
```

Comment:
This surely isn't correct, but I also don't know what to do with it

Reply:
Why so? the field `data` exists
https://github.com/getsentry/sentry-cocoa/blob/be47c6c37e25c3c5d92cceb875c6f408f76017f8/Sources/Sentry/Public/SentryUser.h#L34-L37

Reply:
Yes, but `extras` is also written into `data` and it will then get overwritten by `data` from the dart layer

Reply:
I guess we should merge both and set them to `data` only.